improver_tests/acceptance/SHA256SUMS

```diff
@@ -808,41 +807,41 @@ ea67ae7a7f5363ae3692b29c93fb9989449d96e49dc613a1297d98ddb12c578a  ./wxcode-modal
+766ff2ad1f97d1395c62a32125063d63268c768fd43d572d99f6784e6a62b99f  ./wxcode/global/probability_of_lwe_thickness_of_precipitation_amount_in_vicinity_above_threshold.nc
+7c9c4e9cef368aa60795ae74c22ecf0e012b26afb6c0d1512bde5a403fbc7580  ./wxcode/global/probability_of_lwe_thickness_of_sleetfall_amount_above_threshold.nc
+e6590938ca11473cc2427df9537dee03801a692b63c8b09a191142b951098e19  ./wxcode/global/probability_of_lwe_thickness_of_snowfall_amount_above_threshold.nc
+d0022b02c74ed402c980e179547762f5058f0d3215a5415bcd5b06a50d880fc2  ./wxcode/global/probability_of_number_of_lightning_flashes_per_unit_area_in_vicinity_above_threshold.nc
+ecbaba5b0a467f2b60a12cc00f1365567d9340ff5d8418ab67d24a15c6ee3dd2  ./wxcode/global/probability_of_shower_condition_above_threshold.nc
+9f561f34e871bf3858bbce6e0757d4480898849340192d6e2df9cbdc01d5c01a  ./wxcode/global/probability_of_thickness_of_rainfall_amount_above_threshold.nc
+0d4634a8364ed05ae47249311bb182c8a873d8c36399ad7e647002ea2e2887f3  ./wxcode/global/probability_of_visibility_in_air_below_threshold.nc
+81740bda1027473ee40526f5d8fdc121fc3813f3f5ba68364eefecddcf4ebcfb  ./wxcode/native_units/probability_of_low_and_medium_type_cloud_area_fraction_above_threshold.nc
+ce4f90cfa139e0787c79ec53e5347883f70f4b7a24aeebb0e4fd135de218592f  ./wxcode/native_units/probability_of_low_type_cloud_area_fraction_above_threshold.nc
+39acd5ab43a92defdcad8415dc59797d4465ed9e61add4fb9b2520acec4d9eec  ./wxcode/native_units/probability_of_lwe_graupel_and_hail_fall_rate_in_vicinity_above_threshold.nc
+0772cf9339e7e5c880d8488d6a74980376be8796a89c42305e6539a55d838f2d  ./wxcode/native_units/probability_of_lwe_thickness_of_graupel_and_hail_fall_amount_above_threshold.nc
+2e34db3d4f5f4f22ee11af1be9bb726dad77b25691cca725b4b56f6b305b6df1  ./wxcode/native_units/probability_of_lwe_thickness_of_precipitation_amount_above_threshold.nc
+ef3bd0d3350298e51959184d3b91d94c358a546966a3f13b3caaccaec2e371fb  ./wxcode/native_units/probability_of_lwe_thickness_of_precipitation_amount_in_vicinity_above_threshold.nc
+a148ad61fd67b869c13331c8efd417dc7e0fab2a4bf620c2e9195c4f774bfd14  ./wxcode/native_units/probability_of_lwe_thickness_of_sleetfall_amount_above_threshold.nc
+de6fae5c018225be05851074ce556183e4e82737ade623d41f3435dab0938113  ./wxcode/native_units/probability_of_lwe_thickness_of_snowfall_amount_above_threshold.nc
+a89f2d2781b784612ae1b133cc4044c1b5d4500cbe294639f0549850d8500ab0  ./wxcode/native_units/probability_of_number_of_lightning_flashes_per_unit_area_in_vicinity_above_threshold.nc
+696e6ad5acb0c4245e7d387ab405fcfe395a3266eb498adc8a533276d0cbbe5d  ./wxcode/native_units/probability_of_shower_condition_above_threshold.nc
+88406e9b034f6e2a003935f8cd21a378a525808dae9ba7636b483b7d2b710098  ./wxcode/native_units/probability_of_thickness_of_rainfall_amount_above_threshold.nc
+be903f357d5d2405e36cfbdd57957705b60b64a157c2341f3aeb9001463d000c  ./wxcode/native_units/probability_of_visibility_in_air_below_threshold.nc
```

Comment:
As a note, it would be good to review the suitability of the `wx_decision_tree.json` file prior to making this more widely available.